src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -389,9 +414,9 @@ internal static async Task CheckNonOptionalSettings(IEnumerable<KeyValuePair<str
 
                 if (string.IsNullOrWhiteSpace(azureWebJobsStorage) && !allNonStorageTriggers)
                 {
-                    throw new CliException($"Missing value for AzureWebJobsStorage in {SecretsManager.AppSettingsFileName}. " +
+                    throw new CliException($"Missing value for AzureWebJobsStorage in {andConfigSources}. " +
```

Comment:
nit: move exception message string to constants. This will avoid defining variables orConfigSources / andConfigSources